PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+Fixes #
+
+## Description
+
+- 
+-
+
+@neontribe/contemplating-action-dev
```

Comment:
I'll set this up as a team.

Reply:
I've just shortened it to contemplating-action without the dev. Can you please change here too?